src/components/app/details/appDetails/DeploymentStatusCard.tsx

```diff
@@ -18,65 +25,94 @@ function DeploymentStatusCard({ deploymentStatusDetailsBreakdownData, loadingRes
-                    </Tippy>
-                </div>
-                <div className="flexbox">
-                    <span
-                        data-testid="deployment-status-name"
-                        className={`app-summary__status-name fs-14 mr-8 fw-6 f-${deploymentStatusDetailsBreakdownData.deploymentStatus?.toLowerCase()} ${
-                            deploymentStatusDetailsBreakdownData.deploymentStatus === DEPLOYMENT_STATUS.INPROGRESS
-                                ? 'dc__loading-dots'
-                                : ''
-                        }`}
-                    >
-                        {deploymentStatusDetailsBreakdownData.deploymentStatusText}
-                    </span>
-                    <div className={`${deploymentStatusDetailsBreakdownData.deploymentStatus} icon-dim-20 mt-2`}></div>
-                </div>
-                <div>
-                    <span data-testid="deployment-status-deatils" className="cb-5 fw-6 pointer">
-                        Details
-                    </span>
-                </div>
-            </div>
+            {!hideDeploymentStatusLeftInfo && renderDeploymentStatus()}
             <div className="flex left column mw-140">
-                <div className="fs-12 fw-4 cn-9">Deployment triggered</div>
+                <div className="fs-12 fw-4 cn-9">
+                    {hideDeploymentStatusLeftInfo ? 'Last updated' : 'Deployment triggered'}
+                </div>
                 <div className="flexbox" data-testid="last-updated-time">
                     <span className="fs-13 mr-5 fw-6 cn-9">
-                        {deploymentStatusDetailsBreakdownData.deploymentTriggerTime
-                            ? moment(
+                        {hideDeploymentStatusLeftInfo
+                            ? deploymentTriggerTime && moment(deploymentTriggerTime, 'YYYY-MM-DDTHH:mm:ssZ').fromNow()
+                            : deploymentStatusDetailsBreakdownData?.deploymentTriggerTime &&
+                              moment(
                                   deploymentStatusDetailsBreakdownData.deploymentTriggerTime,
                                   'YYYY-MM-DDTHH:mm:ssZ',
-                              ).fromNow()
-                            : '-'}
+                              ).fromNow()}
                     </span>
-                    {deploymentStatusDetailsBreakdownData.deploymentStatus === DEPLOYMENT_STATUS.INPROGRESS && (
+                    {deploymentStatusDetailsBreakdownData?.deploymentStatus === DEPLOYMENT_STATUS.INPROGRESS && (
                         <Timer className="icon-dim-16 mt-4" />
                     )}
                 </div>
-                <div className="fw-4 fs-12 cn-9 dc__ellipsis-right dc__mxw-inherit">
-                    by {deploymentStatusDetailsBreakdownData.triggeredBy || '-'}
-                </div>
+
+                {hideDeploymentStatusLeftInfo
+                    ? triggeredBy && triggeredBy
```

Comment:
why are using and operation between same variable

Reply:
fixed